how_to_build.md

```diff
@@ -9,17 +9,22 @@ At the time of writing the build is only confirmed to work on Windows using the
 
 ## Prerequisites
 
-1. Ensure you have .NET framework 4.6.1 or later installed.
+The build requires that a few pieces of software be installed on the host computer. We're somewhat aggressive about adoptiong new language features and the like, so rather than specifying exactly which versions are required, we'll tend toward
+"latest" or "at least" forms of guidance. If it seems you have an incompatible version of the software, prefer to upgrade rather than downgrade.
 
-1. Ensure you have version 15.3 or later of either Visual Studio 2017 or MSBuild installed.
+Ensure that the following are installed:
 
-1. Ensure you have .NET Core SDK 2.0.2 or later installed
+1. .NET framework 4.6.1 or later

```

Comment:
I think we also need the 4.6.1 targeting pack, not just the runtime. (but it's installed with VS anyway, so maybe not worth mentioning)

Reply:
Is it installed with MSBuild?

Reply:
No idea. I don't even know how to install MSBuild without installing VS...

Reply:
Build from source? ;)

Reply:
Actually, there's that thing called "Build tools for Visual Studio", intended for build servers which don't need the IDE itself. I think it includes the targeting packs.
Anyway, it's probably not an issue. Forget I ever mentioned it :wink: